common/network-common/src/main/java/org/apache/spark/network/crypto/TransportCipher.java

```diff
@@ -203,6 +203,63 @@ public long transfered() {
       return transferred;
     }
 
+    @Override
```

Comment:
I prototyped this change internally for a different reason, and I chose to create an abstract class for all this boilerplate, which seems cleaner to me.

```
public abstract class AbstractFileRegion extends AbstractReferenceCounted implements FileRegion {

  @Override
  @SuppressWarnings("deprecation")
  public final long transfered() {
    return transferred();
  }

  @Override
  public FileRegion retain() {
    super.retain();
    return this;
  }

  @Override
  public FileRegion retain(int increment) {
    super.retain(increment);
    return this;
  }

  @Override
  public FileRegion touch() {
    return touch(null);
  }

  @Override
  public FileRegion touch(Object hint) {
    return this;
  }

}
```

Reply:
Good point! Updated!